src/Controller/UserController.php

```diff
@@ -164,27 +205,40 @@ public function modifyUser(User $user, $userId, UserRepository $repository, Enti
         $manager->persist($registeredUser);
         $manager->flush();
 
-        return $this->view($registeredUser, Response::HTTP_ACCEPTED);
+        $keyAll = 'user.all?page=' . $request->query->getInt('page', 1);
+        $keyOnce = 'user.once';
+
+        $cacheAll = $this->adapter->getItem($keyAll);
+        $cacheOnce = $this->adapter->getItem($keyOnce);
+
+        if (true === $cacheAll->isHit()){
+            $this->adapter->clear();
+        } elseif (true === $cacheOnce->isHit()) {
+            $this->adapter->clear();
+        }
+
+        return $this->view($registeredUser);
```

Comment:
grace au @view, tu peux juste retouner le $registeredUser

Reply:
I have an other return with a view in this fonction.
If I return only the object, PHPStorm mentions a mistake